### PR TITLE
Feature/api 189

### PR DESCRIPTION
**Pull request by moniquejones08**

### CODE REVIEW

**Comment by moniquejones08** [comment]
@stuisme @BrandonEMiller @brianhyder

File: apiary.apib [hunk 1]
```diff
@@ -328,6 +347,11 @@ Represents a user with all properties included.
 + `updatedDateTime`: `2015-01-01T00:00:00.123Z` (string, optional) - The UTC date and time the object was last updated.
 + `updatedByApiClient` (Base Client, required) - The API Client that last updated the object.
 
+## User Activation (User Detail)
```
Comment (larastocks) [inline apiary.apib:358]: Does this need to be broken up into description and properties?

File: apiary.apib [hunk 2]
```diff
@@ -320,6 +332,13 @@ Represents a user with all properties included.
 
 ### Properties
 + `hourlyRate`: 25.50 (number, optional) - The hourly Rate
++ `roles` (array[Security Role], required) - The collection of roles associated with the user.  When authenticating against an application, only the roles for the user in the context of that application will be returned.
++ `isActivated`: true (boolean, required) - Indicates if a user has been activated within the platform
++ `isEnabled`: true (boolean, required) - Indicates if the user is able to login to the platform
+
+## User Detail (User)
```
Comment (larastocks) [inline apiary.apib:345]: Does this need to be broken up into description and properties?

Comment (moniquejones08) [inline apiary.apib:345]: Facts! I totally missed that when I pulled it into my branch.  Thanks!

File: apiary.apib (hunk 1 above)
Comment (moniquejones08) [inline apiary.apib:358]: Facts! I totally missed that when I pulled it into my branch.  Thanks!

File: apiary.apib (hunk 2 above)
Comment (moniquejones08) [inline apiary.apib:345]: Fixed

File: apiary.apib (hunk 1 above)
Comment (moniquejones08) [inline apiary.apib:358]: Fixed

**Comment by larastocks** [comment]
+1

File: apiary.apib (hunk 1 above)
Comment (brianhyder) [inline apiary.apib:358]: Thanks @moniquejones08, I'm failing at life today.

**Pull request merged**